src/js/util.js

```diff
@@ -1052,6 +1052,10 @@
             } else {
                 el.parentNode.removeChild(el);
             }
+        },
+
+        toArr: function (arraylike) {
+            return [].slice.call(arraylike);
```

Comment:
Is there a benefit to calling `[].slice()` as opposed to `Array.prototype.slice`?